Recipe for target `<package>-server.pot` failed if no i10n messages

Project:

* GeoMapFish version: **2.1.2**

Running command: `make -f instance_v2.mk upgrade4 `

Result:

```
.build/venv/bin/c2ctool upgrade  instance_v2.mk --step 4
make[1]: Entering directory 'v2'
.build/venv/bin/pot-create --keyword _ --config lingua-server.cfg --output nyon/locale/nyon-server.pot nyon/models.py nyon/templates/desktop.html nyon/templates/mobile.html nyon/templates/api/xapihelp.html nyon/templates/api/apihelp.html nyon/templates/api/apihelp_old.html
v2/.build/venv/local/lib/python2.7/site-packages/lingua/extract.py:330: UnicodeWarning: Python extractor called with bytes input. Please update your plugin to submit unicode instead.
  for message in extractor(real_filename, options):
No translatable strings found, aborting
CONST_Makefile:698: recipe for target 'nyon/locale/nyon-server.pot' failed
make[1]: *** [nyon/locale/nyon-server.pot] Error 2
make[1]: Leaving directory 'v2'
Traceback (most recent call last):
  File ".build/venv/bin/c2ctool", line 11, in <module>
    sys.exit(main())
  File "v2/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 100, in main
    c2ctool.upgrade()
  File "v2/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 231, in upgrade
    self.step4()
  File "v2/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 456, in step4
    check_call(["make", "-f", self.options.file, "build"])
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-f', 'instance_v2.mk', 'build']' returned non-zero exit status 2
CONST_Makefile:1288: recipe for target 'upgrade4' failed
make: *** [upgrade4] Error 1
```

--- a/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
+++ b/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
@@ -697,11 +697,11 @@ $(PACKAGE)/locale/$(PACKAGE)-$(L10N_SERVER_POSTFIX).pot: \
 		$(SERVER_LOCALISATION_SOURCES_FILES)
 	$(PRERULE_CMD)
 	rm -f $@ # Because of WindowsError, thus only problematic on Windows
-	$(VENV_BIN)/pot-create --keyword _ --config $< --output $@ $(SERVER_LOCALISATION_SOURCES_FILES)
+	$(VENV_BIN)/pot-create --keyword _ --config $< --output $@ $(SERVER_LOCALISATION_SOURCES_FILES) || true
 	# removes the always changed date line
-	sed -i '/^"POT-Creation-Date: /d' $@
-	sed -i '/^"PO-Revision-Date: /d' $@
-	chmod go+r $@
+	[ ! -f $@ ] || sed -i '/^"POT-Creation-Date: /d' $@
+	[ ! -f $@ ] || sed -i '/^"PO-Revision-Date: /d' $@
+	[ ! -f $@ ] || chmod go+r $@
 
 .PRECIOUS: $(PACKAGE)/locale/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).pot
 $(PACKAGE)/locale/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).pot: \
@@ -725,7 +725,7 @@ $(PACKAGE)/locale/%/LC_MESSAGES/$(PACKAGE)-$(L10N_SERVER_POSTFIX).po: $(PACKAGE)
 	mkdir -p $(dir $@)
 	[ -f $@ ] || msginit --no-translator --input=$< --output-file=$@ -l $*
 	touch --no-create $@
-	msgmerge --backup=none --update --sort-output --no-location $@ $<
+	[ ! -f $< ] || msgmerge --backup=none --update --sort-output --no-location $@ $<
 
 .PRECIOUS: $(PACKAGE)/locale/%/LC_MESSAGES/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).po
 $(PACKAGE)/locale/%/LC_MESSAGES/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).po: $(PACKAGE)/locale/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).pot

